.pre-commit-config.yaml

```diff
@@ -9,9 +9,11 @@ repos:
     rev: 22.8.0
     hooks:
     -   id: black
+        args: [--line-length=79]
         exclude: ^docs/
-- repo: https://github.com/pycqa/isort
-  rev: 5.13.1
-  hooks:
-  - id: isort
-    name: isort (python)
+-   repo: https://github.com/pycqa/isort
+    rev: 5.13.1
+    hooks:
+    -   id: isort
+        name: isort (python)
+        args: [--profile=black, --line-length=79]
```

Comment:
Happy to use whatever method you prefer since this is effectively your repo now - but just so you know we use the [`pyproject.toml`](https://github.com/catalystneuro/jazayeri-lab-to-nwb/blob/watters/pyproject.toml#L24) to configure things like this

Reply:
Thanks!  I've updated pyproject.toml.